twitter_clonev2/settings.py

```diff
@@ -48,6 +48,10 @@
 REST_FRAMEWORK = {
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
     'PAGE_SIZE': 5,
+    # 'DEFAULT_AUTHENTICATION_CLASSES': (
+    #     'tweetapi.authentication.Authentication',
+    # ),
+
```

Comment:
You can set the ones you don't want this to affect to None or

`permission_classes = []`

Reply:
fixed sir